README.md

```diff
@@ -72,4 +75,40 @@ auto model = DetectionModel::create_model("\~/.cache/omz/public/ssd300/FP16/ssd3
  }
 ```
 
-More examples of how to use asynchronous inference and serving are coming soon.
\ No newline at end of file
+For more details please refer to the [examples](https://github.com/openvinotoolkit/model_api/tree/master/examples) of this project.
+
+## Supported models
+### Python:
+- Image Classification:
+  - All the [OpenVINO Model Zoo models](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/index.md#classification-models)
+  - Any model converted to OpenVION IR format
```

Comment:
```suggestion
```
`Any model converted to OpenVION IR format` is a very general statement. Maybe we could just not specify it at all